packages/terra-dropdown-button/src/_Dropdown.jsx

```diff
@@ -26,14 +26,6 @@ const propTypes = {
    * Width to set the dropdown to. Used when `isBlock` is true. Unset means to autosize.
    */
   width: PropTypes.string,
-  /**
-   * Whether or not dropdown is opened using keyboard.
-   */
-  openedViaKeyboard: PropTypes.bool,
-  /**
-   * Callback for reference of the dropdown button
-   */
-  buttonRef: PropTypes.func,
```

Comment:
Why are these props are being removed? This would be a breaking change which would break backwards compatibility and break consumers' builds.

Reply:
Since these props are not created / exposed for consumers removal of these prop would require jest snapshot updates. Will this be considered as Breaking Change ..??

Reply:
Oh okay. I now see that that these were never exposed to consumers so that's fine then. Thanks for clarifying!